ui/partials/perf/comparectrl.html

```diff
@@ -4,8 +4,9 @@
       Loading all results, please wait a minute...
       <img src="img/dancing_cat.gif" />
     </div>
-    <div id="error" ng-if="!dataLoading && errors.length" ng-repeat="error in errors">
-      <div>{{error}}</div>
+    <div id="error" ng-if="!dataLoading && errors.length">
+      <p ng-repeat="error in errors" class="bg-danger">{{error}}</p>
+      <a href="#/comparechooser?originalProject={{originalProject}}&originalRevision={{originalRevision}}&newProject={{newProject}}&newRevision={{newRevision}}">Modify choice</a>
```

Comment:
This is good, but I think this could look a bit better, now that I think about it.

Let's make the containing div an "alert":

http://getbootstrap.com/components/#alerts

i.e. add `class="alert alert-danger" role="alert"` to it.

Then let's add an error icon for each error, as demonstrated here:

http://getbootstrap.com/components/#glyphicons-examples

(note the `bg-danger` class is no longer needed)

We could make this error summary a directive and reuse it in both comparectrl and comparesubtestctrl, as you were elsewhere. That way we don't need to repeat code.